### PR TITLE
feat(home): implement data reload on logo click #2952

### PR DESCRIPTION
**Pull request by martinchmui**
This pull request introduces functionality to reload data in the **`HomeOverviewComponent`** when the Ghostfolio logo is clicked. It aims to enhance user experience by allowing users to easily refresh their data without needing to navigate away or manually refresh the entire page. The implementation listens for click events on the logo and triggers the data fetch method already in use within the component. This feature is a direct response to Issue #2952.

### CODE REVIEW

File: apps/client/src/app/components/home-overview/home-overview.component.ts [hunk 1]
```diff
@@ -35,13 +36,15 @@ export class HomeOverviewComponent implements OnDestroy, OnInit {
   public showDetails = false;
   public unit: string;
   public user: User;
+  private subscription: Subscription;
```
Comment (dtslvr) [inline apps/client/src/app/components/home-overview/home-overview.component.ts:39]: Remove

File: apps/client/src/app/components/home-overview/home-overview.component.ts [hunk 2]
```diff
@@ -101,6 +108,9 @@ export class HomeOverviewComponent implements OnDestroy, OnInit {
   public ngOnDestroy() {
     this.unsubscribeSubject.next();
     this.unsubscribeSubject.complete();
+    if (this.subscription) {
```
Comment (dtslvr) [inline apps/client/src/app/components/home-overview/home-overview.component.ts:111]: Remove

File: apps/client/src/app/core/layout.service.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class LayoutService {
+  private shouldReloadSubject = new Subject<void>();
+
+  // Observable stream
+  public shouldReload$ = this.shouldReloadSubject.asObservable();
+
+  constructor() {}
```
Comment (dtslvr) [inline apps/client/src/app/core/layout.service.ts:13]:
```
public constructor() {}
```

File: apps/client/src/app/components/home-overview/home-overview.component.ts [hunk 4]
```diff
@@ -35,13 +36,15 @@ export class HomeOverviewComponent implements OnDestroy, OnInit {
...
   public user: User;
+  private subscription: Subscription;
 
   private unsubscribeSubject = new Subject<void>();
 
   public constructor(
     private changeDetectorRef: ChangeDetectorRef,
     private dataService: DataService,
     private deviceService: DeviceDetectorService,
+    private layoutService: LayoutService,
```
Comment (dtslvr) [inline apps/client/src/app/components/home-overview/home-overview.component.ts:47]: Please move down to keep the alphabetical order

File: .vscode/settings.json [hunk 5]
```diff
@@ -1,4 +1,5 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "angular.enable-strict-mode-prompt": false
```
[the remark this answers is not in the document]
Comment (martinchmui) [reply .vscode/settings.json]: Unintended. Probably some settings with my VSCode.

**Review by dtslvr — APPROVED** [review]
Thanks again, @martinchmui! Ready to merge now.

**Pull request merged**